hypertrace-trace-enricher/enriched-span-constants/src/main/java/org/hypertrace/traceenricher/enrichedspan/constants/EnrichedSpanConstants.java

```diff
@@ -17,7 +17,7 @@ public class EnrichedSpanConstants {
   public static final String UNIQUE_API_NODES_COUNT = "unique.apis.count";
   public static final String GRPC_REQUEST_URL = "grpc.request.url";
   public static final String GRPC_REQUEST_ENDPOINT = "grpc.request.endpoint";
-  public static final String INTERNAL_SVC_LATENCY = "enriched.serviceInternalProcessingTime";
+  public static final String INTERNAL_SVC_LATENCY = "enriched.internalDurationMillis";
```

Comment:
Umm... seems like users might not be able to understand the meaning of the metric from the new name.

Reply:
Also, tag keys should be in lowercase as this is the convention being followed in Hypertrace.

Reply:
Addressed.